packages/atomic/src/components/common/atomic-modal/atomic-modal.tsx

```diff
@@ -76,7 +77,9 @@ export class AtomicModal implements InitializableComponent<AnyBindings> {
       if (watchToggleOpenExecution !== this.currentWatchToggleOpenExecution) {
         return;
       }
-      this.focusTrap!.active = true;
+      if (!this.noFocusTrap) {
+        this.focusTrap!.active = true;
+      }
```

Comment:
You could check for the absence of the ref instead.
```suggestion
      this.focusTrap && (this.focusTrap.active = true);
```

If you're in too much of a hurry, you can merge anyway

Reply:
Ah I didn't know you guys accepted this kind of shorthand. I personally like it but sometimes people find it harder to read.